tower-balance/src/p2c/service.rs

```diff
@@ -132,7 +132,7 @@ where
                 .next_ready_index
                 .and_then(|i| Self::repair_index(i, idx, self.ready_services.len()));
             debug_assert!(!self.cancelations.contains_key(key));
-        } else if let Some(cancel) = self.cancelations.remove(key) {
+        } else if let Some(cancel) = self.cancelations.swap_remove(key) {
```

Comment:
does this have the same affect? Why this change?

Reply:
`remove` is deprecated, so our `#![deny(warnings)]` gives a compilation failure. According to the docs, `remove` was the same as `swap_remove`. I imagine they're just clarifying the naming.

Reply:
Yup just checked 👍